decidim-comments/lib/decidim/comments/engine.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+require "rails"
+require "active_support/all"
+
+require "decidim/core"
+require "jquery-rails"
+require "sass-rails"
+require "turbolinks"
+require "foundation-rails"
+require "foundation_rails_helper"
+require "react-rails"
+
+module Decidim
+  module Comments
+    # Decidim's core Rails Engine.
+    class Engine < ::Rails::Engine
+      isolate_namespace Decidim::Comments
+
+      initializer "decidim_comments.assets" do |app|
+        app.config.assets.precompile += %w(decidim_comments_manifest.js)
```

Comment:
Take as an example `decidim-admin`. By including assets in `decidim_comments_manifest` (which is currently empty), sprockets will generate a separate comments bundle.

you can then just require the JS when needed on the comments helper via que `javascript_include_file` helper, thus allowing lazy-loading of the script (instead of bundling it into `decidim.js` which is where currently is required via sprockets).